npm-shrinkwrap.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "generator-assets",
-  "version": "2.5.0-dev",
+  "version": "2.3.0-dev",
```

Comment:
I manually incremented it for 0.5 and again for 0.6 since it was incremented for 0.4.